src/pep8_naming/checks.rs

```diff
@@ -215,9 +276,31 @@ pub fn camelcase_imported_as_acronym(
     None
 }
 
+pub fn error_suffix_on_exception_name(
+    class_def: &Stmt,
+    bases: &[Expr],
+    name: &str,
+) -> Option<Check> {
+    if bases.iter().any(|base| {
+        if let ExprKind::Name { id, .. } = &base.node {
+            id == "Exception"
+        } else {
+            false
+        }
```

Comment:
The original implementation checks all the base classes:
https://github.com/PyCQA/pep8-naming/blob/main/src/pep8ext_naming.py#L302